components/HomePage.jsx

```diff
@@ -40,6 +120,13 @@ export default HomePage = ({ navigation }) => {
         }}
         title="View sightings"
       />
+
+      {user && (
```

Comment:
I didn't see this sign-out button after logging in - do we need to force a refresh or something to show the signout button? I still see the login button after signing up

Reply:
yes we need to refresh the code on HomePage and it will automatically save the currentUser